dev/constraints.h

```diff
@@ -440,6 +443,12 @@ namespace sqlite_orm {
         template<class T>
         SQLITE_ORM_INLINE_VAR constexpr bool is_generated_always_v = is_generated_always<T>::value;
 
+        template<class T>
```

Comment:
Let's remove those traits as they are currently superfluous? Keep in sync with other constraints that are only used in a single place like `unique_t` or `collate_constraint_t`.

Reply:
done